test/index.js

```diff
@@ -0,0 +1,251 @@
+/* eslint-disable no-unused-expressions */
+const fs = require('fs-extra');
+const { expect } = require('chai');
```

Comment:
I recommend using `require('assert').strict` instead. Please refer to `Engineering Huddle: 2022-08-30` in notion

Reply:
Done